CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# v0.5.0 (November 2020)
+- Various general (but very important) speed improvements [[#96]](https://github.com/aiidateam/disk-objectstore/pull/96) [[#102]](https://github.com/aiidateam/disk-objectstore/pull/102)
+- Add callbacks to a number of functions (e.g. export, add_objects_to_pack, ... to allow showing progress bars or similar indicators [[#96]](https://github.com/aiidateam/disk-objectstore/pull/96)
+- Implement repacking (at least when not changing hashing or compression) [[#96]](https://github.com/aiidateam/disk-objectstore/pull/96)
+- Remove `export` function, implement `import_objects` function instead, to be called on the other side (it's more efficient) [[#96]](https://github.com/aiidateam/disk-objectstore/pull/96)
+- Add support for VACUUMing operations on the SQLite database (very important for efficiency) [[#96]](https://github.com/aiidateam/disk-objectstore/pull/96)
```

Comment:
Just a quick check that it is intended that all of these point to the same PR. Maybe all these points were all addressed in the same. If this is OK, then it looks good for me to merge

Reply:
Hi, yes. In the end I started addressing one issue, and one thing lead to another, and I ended up with a single "monster" PR - I know it's not a good idea in general but some of the features where quite interconnected.